client/src/modules/list/model/selectors.js

```diff
@@ -106,5 +106,5 @@ export const getCohortArchivedLists = createSelector(
 
 export const getMembers = createSelector(
   getList,
-  list => list && _keyBy(list.members, '_id')
+  list => list && list.members
```

Comment:
same issue as with cohort

Reply:
without this check we get an error: Uncaught TypeError: Cannot read property 'members' of undefined